pkg/utils/etcdutil/etcdutil.go

```diff
@@ -41,6 +41,13 @@ const (
 	// defaultAutoSyncInterval is the interval to sync etcd cluster.
 	defaultAutoSyncInterval = 60 * time.Second
 
+	// defaultDialKeepAliveTime is the time after which client pings the server to see if transport is alive.
+	defaultDialKeepAliveTime = 10 * time.Second
```

Comment:
What is the default value if we don't set them?

Reply:
zero